train.py

```diff
@@ -193,6 +194,7 @@
     optimizer.load_state_dict(checkpoint["optimizer"])
 checkpoint = None  # free up memory
 
+compile = compile if device != "mps" else False
```

Comment:
Does it need to be "pythonic"?

Isn't this clearer?

```python
if device == "mps": compile = False
```

If you guys really prefer "pythonic", there is this option also:

```python
compile = False if device == "mps" else compile
```